packages/translations/scripts/extract-string.js

```diff
@@ -0,0 +1,39 @@
+const glob = require('glob');
+const path = require('path');
+
+const getRegexPattern = () => new RegExp(/i18n_default_text=(['"])(.*?)(?<!\\)\1|localize\(\s*(['"])\s*(.*?)\s*(?<!\\)\3\s*/gs);
+
+const getStringsFromInput = (input, i18n_marker = getRegexPattern()) => {
+    const messages = [];
+    while ((result = i18n_marker.exec(input)) !== null) {
+        // If it captures `text=` then it will be index 2, else its index 4 which captures `localize`
+        const extracted = result[2] || result[4]
+        // Replace escape characters.
+        messages.push(extracted.replace(/\\/g, '')); 
+    }
+    return messages;
+}
+
+const getTranslatableFiles = () => {
+    const packages_with_translations = ['bot-skeleton', 'bot-web-ui', 'trader', 'core'];
+    const globs = ['**/*.js', '**/*.jsx'];
+    const file_paths = [];
+
+    // Bot: Find all file types listed in `globs`
```

Comment:
Looks very good, thanks @aaron-binary

Could we just remove this old comment too since it's outdated

Reply:
Sure thing 👍